onchain/cairo/afk/src/bip340.cairo

```diff
@@ -565,35 +611,35 @@ mod tests {
 
         assert!(verify(px, rx, s, m.into()));
     }
-    #[test]
-    fn test_20() {
-        let (private_key, public_key) = generate_keypair();
+    // #[test]
+    // fn test_20() {
+    //     let (private_key, public_key) = generate_keypair();
```

Comment:
Please uncomment this test and run your tests. 
Because your test doesn't work, change the address in the input with the correct one your used in your fork test please